protostar-rust/tests/data/dispatchers/tests/using_dispatchers.cairo

```diff
@@ -15,7 +15,7 @@ use dispatchers::hello_starknet::IHelloStarknetDispatcherTrait;
 #[test]
 fn call_and_invoke() {
     let class_hash = declare('HelloStarknet').unwrap();
-    let prepared = PreparedContract { contract_address: 1234, class_hash: class_hash, constructor_calldata: @ArrayTrait::new() };
+    let prepared = PreparedContract { contract_address: 1234, class_hash: class_hash, constructor_calldata: ArrayTrait::new().span() };
```

Comment:
I can change it here, it is quite easy to do. I agree since the calldata is not used for anything but creating the `PreparedContract` struct, therefore we can make the struct consume the calldata